wandb/apis/public/files.py

```diff
@@ -131,6 +132,20 @@ def size(self):
             return int(size_bytes)
         return 0
 
+    @property
+    def path_uri(self) -> str:
+        """
+        Returns the uri path to the file in the storage bucket.
+        """
+        path_uri = ""
+        try:
+            path_uri = utils.parse_s3_url_to_s3_uri(self._attrs["directUrl"])
```

Comment:
if we want to extend this to Google and Azure, we just add different parsing check here